django_spellbook/spellblocks.py

```
@@ -300,4 +302,114 @@ def get_context(self):
+        except TypeError: # Handles if value_str is None or other non-string/non-numeric type
+            logger.error(f"Invalid type for dimension value: '{value_str}'. Using default: '{default_value_with_unit}'.")
+            return default_value_with_unit
+
+
+    def get_context(self):
+        context = super().get_context()
+
+        # Position
+        pos = self.kwargs.get("pos", self.DEFAULT_POS).lower()
+        if pos not in ["start", "center", "end"]: # These correspond to flex justify-content values
+            logger.error(f"AlignBlock: Invalid 'pos' parameter: '{pos}'. Defaulting to '{self.DEFAULT_POS}'.")
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)